submissions/OnyediriConstancia/css/css/product.css

```diff
@@ -0,0 +1,319 @@
+body {
```

Comment:
You only need to create a single `css` folder. But, here I can see you create a `css` folder inside another `css` folder